mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule ElixirTools.MixProject do
   use Mix.Project
 
-  @version "0.6.0"
+  @version "0.7.0"
```

Comment:
never bump the version in the request itself, this can cause weird issues as a PR like this can stay open longer